app/src/main/java/com/example/mystore/ui/home/layouts/ProductRow.kt

```diff
@@ -75,27 +73,19 @@ fun ProductRow(
                 RatingBar(
                     modifier = Modifier
                         .fillMaxWidth()
-                        .padding(end= dimensionResource(id = R.dimen.padding_large)),
+                        .padding(end= dimensionResource(id = R.dimen.padding_right_rating)),
                     rating = product.rating
                 )
 
                 Text(
-                    text = NumberFormat.getCurrencyInstance(Locale("en", "US"))
+                    text = NumberFormat.getCurrencyInstance()
```

Comment:
I might have mislead you. `Locale.US` (better syntax for `"en", "US"`) might indeed be needed as we're using the dollar currency.

Also, it might be cleaner to extract this as a `Double` extension function.